shiny_app/server.R

```diff
@@ -351,4 +351,204 @@ shinyServer(function(input, output, session) {
       })
     })
 
+
+  ## Surge Tabs ####
+
+  summary_outputs <- reactive({
+    model_output() %>%
+      filter(near(time, round(time))) %>%
+      group_by_at(vars(time:treatment)) %>%
+      summarise_all(sum) %>%
+      rename(treatment_pathway = treatment)
+  })
+
+  ## Tab - Subpopn ####
+
+  output$surge_subpopn <- renderTable({
+    summary_outputs() %>%
+      pivot_wider(names_from = type, values_from = value) %>%
+      dplyr::group_by(group) %>%
+      filter(!is.na(group)) %>%
+      summarise(`Adjusted exposed / at risk @ baseline` = as.integer(round(sum(`new-at-risk`, na.rm = T), 0)),
+                `Total symptomatic over period referrals` = as.integer(round(sum(`new-referral`, na.rm = T), 0)),
+                `Total receiving services over period` = as.integer(round(sum(`new-treatment`, na.rm = T), 0))) %>%
+      arrange(-`Total symptomatic over period referrals`) %>%
```

Comment:
I would generally suggest using `arrange(desc(column))` vs `arrange(-column)` due to the confusion that can be caused with `select(-col)` etc.